packages/boot/test/upgrading/test-upgrade-vats.js

```diff
@@ -74,7 +74,7 @@ test('upgrade vat-board', async t => {
   const { bfile } = t.context;
   const bundles = {
     board: {
-      sourceSpec: bfile('../../src/vat-board.js'),
+      sourceSpec: bfile('../../../vats/src/vat-board.js'),
```

Comment:
I'm not very comfortable with a relative path reaching out of a package into another one.  IIUC, everything here that is a `sourceSpec` already accepts specifiers like `'@agoric/vats/...'`.  If this isn't worth investigating and cleaning up right now, maybe just place a `TODO` near `bfile`'s definition that it and relative paths may not be necessary.

Reply:
there's quite a precedent for this: https://github.com/Agoric/agoric-sdk/blob/50332a848618b9bec7fee9f565deea3a1e7827d3/packages/zoe/test/swingsetTests/makeKind/test-makeKind.js#L47

but I'll follow the Scouts code and leave this better.